web/cypress/e2e/pages/people.cy.ts

```diff
@@ -57,3 +57,20 @@ describe("Import approved researchers", () => {
     cy.get("table").contains(username).should("be.visible");
   });
 });
+
+describe("Invite externalss", () => {
+  it("should not be visible to a base user", () => {
+    cy.loginAsBase();
+    cy.visit("/people");
+    cy.get("[data-cy='show-invite-input']").should("not.exist");
+  });
+
+  it("should be visable to & usable by an admin user", () => {
+    cy.loginAsAdmin();
+    cy.visit("/people");
+    cy.get("[data-cy='show-invite-input']").should("be.visible").click();
+
+    cy.get("input[name='email']").should("be.visible").type("lab.amanda.ch@gmail.com");
+    cy.get("[data-cy='send-invite']").should("be.visible").click();
```

Comment:
maybe we don't actually want to send invites in the e2e – I'd probably mock this response. @jhughes982 what do you think?